test/extended/operators/olm.go

```diff
@@ -87,4 +88,19 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			o.Expect(msg).To(o.Equal("IfNotPresent"))
 		}
 	})
+
+	// OCP-21082 - Implement packages API server and list packagemanifest info with namespace not NULL
+	// author: bandrade@redhat.com
+	g.It("Implement packages API server and list packagemanifest info with namespace not NULL", func() {
+		msg, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("packagemanifest", "--all-namespaces", "--no-headers").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		packageserverLines := strings.Split(msg, "\n")
+		if len(packageserverLines) > 0 {
```

Comment:
Don't you need to fail this if the len is zero, not default to passing?

Reply:
If len is zero, the first word is filled in the first character of the line which means that https://bugzilla.redhat.com/show_bug.cgi?id=1670311 is not happening. If it's greater than 0 could be suspicious of a regression.

Reply:
How does it mean that? Msg could be a totally empty string.

Reply:
got your point, will correct that

Reply:
@bparees any concerns about this?

Reply:
you should skip the test in environments where you can't expect testable data.  having a test that defaults to passing in the face of unexpected/unvalidatable results is not a good practice.  It means that code which breaks this behavior can be merged accidentally.

Reply:
Thanks for your feedback. Failing by default as oriented.